packages/grafana-data/src/field/fieldComparers.ts

```diff
@@ -55,7 +55,11 @@ const stringComparer = (a: string, b: string): number => {
   if (!a || !b) {
     return falsyComparer(a, b);
   }
-  return a.localeCompare(b);
+
+  // Using the Intl.Collator object compare method results in much faster string sorting
+  // (estimated 2x as fast) than .localeCompare
+  const compare = new Intl.Collator('en', { sensitivity: 'base' }).compare;
```

Comment:
the trick is to pull this out of the comparison loop. you want to only instantiate this one time globally.